### DISCUSSION TITLE
Milvus Lite in TS

### DISCUSSION THREAD

**Discussion by linonetwo** [milvus-io/milvus#24461] [Ideas & Feature requests]
Can it be used in TS/ Electron?

Some example&instruction is needed.

**Comment by yhmo** [comment]
Milvus Lite is running as a python application.
I am not familiar with ts/electron. I think there are two approaches to build "Milvus Lite + TS" application:
1. Use python FastAPI or Flask to build a backend(a Milvus lite server running in this backend), and use TS/Electron to build the frontend. The frontend call restful api of the backend.
2. Start a Milvus lite server by a python program, and wrap the [Milvus node sdk](https://github.com/milvus-io/milvus-sdk-node) into TS/Electron application to interact with the Milvus server.

**Reply by linonetwo** [reply]
Thanks, but I think I have to use https://www.npmjs.com/package/sqlite-vss instead, which declares nodejs support.
